README.md

```diff
@@ -1,2 +1,70 @@
 # frontend
 The frontend of the MiSArch reference architecture: The web application for the online shop.
+
+## Project setup
```

Comment:
as you decided to yarn, please adapt README

Reply:
Will do